CMakeLists.txt

```diff
@@ -113,6 +134,10 @@ target_include_directories(sonata_static
     PRIVATE $<TARGET_PROPERTY:HighFive,INTERFACE_INCLUDE_DIRECTORIES>
 )
 
+include_directories(${SONATA_INCLUDE_DIR})
+include_directories(${SONATA_SRC_DIR})
```

Comment:
why is a source directory being added as an include dir?

Reply:
needed by tests

Reply:
Refer to above